luigi/scheduler.py

```diff
@@ -758,12 +512,16 @@ def _prune_short_lived_stakeholders(self):
 
     @rpc_method()
     def prune(self):
-        logger.debug("Starting pruning of task graph")
+        logger.info("Starting pruning of task graph")
         self._prune_workers()
+        logger.info("Just pruned workers")
         self._prune_tasks()
+        logger.info("Just pruned tasks")
         self._prune_emails()
+        logger.info("Just pruned emails")
         self._prune_short_lived_stakeholders()
-        logger.debug("Done pruning task graph")
+        logger.info("Just pruned short lived stakeholders")
+        logger.info("Done pruning task graph")
```

Comment:
this will show up in the log a LOT, is this just for testing for now?

Reply:
Whoops, yeah was for debugging!